quickstart/java/jre/virtual-assistant/README.md

```diff
@@ -1,14 +1,14 @@
-# Quickstart: Using virtual assistant (Preview) in Java on Windows or Linux (Ubuntu 16.04, Ubuntu 18.04, Debian 9)
+# Quickstart: Using voice assistant in Java on Windows or Linux (Ubuntu 16.04, Ubuntu 18.04, Debian 9)
 
-This sample demonstrates how to recognize speech for custom voice-first virtual assistants (Preview) and receive activity responses.
-See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-virtual-assistant-java-jre) on the SDK documentation page which describes how to build this sample from scratch in Eclipse.
+This sample demonstrates how to recognize speech for custom voice assistant and receive activity responses.
+See the [accompanying article](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/quickstart-voice-assistant-java-jre) on the SDK documentation page which describes how to build this sample from scratch in Eclipse.
 
 > **Note:**
 > the Speech SDK for the JRE currently supports only the Windows x64 platform, macOS 10.13 or later, and Linux x64 and ARM64 (Ubuntu 16.04, Ubuntu 18.04, Debian 9).
 
 ## Prerequisites
 
-* A subscription key for the Speech service. See [Try the speech service for free](https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started).
+* A subscription key for the Speech service any one of the supported [Voice Assistant Azure regions](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/regions#voice-assistants). See [Try the speech service for free](https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started).
```

Comment:
>en-us [](start = 128, length = 5)

should we have without "en-us" so that page defaults to language selected in customer browser?

Reply:
Yes, good catch, I'll fix it here and in the other 2 PRs.